how-to/customize-workspace/client/src/framework/platform/platform-override.ts

```diff
@@ -292,6 +295,17 @@ export const overrideCallback: WorkspacePlatformOverrideCallback = async (Worksp
 				return super.quit(payload, callerIdentity);
 			}
 		}
+
+		public async handleAnalytics(events: AnalyticsEvent[]) {
+			if (analyticsProvider.isEnabled()) {
+				const platformEvents: PlatformAnalyticsEvent[] = [];
+				const timestamp = new Date().toISOString();
```

Comment:
Prefer Date.now() as a number not a string

Reply:
I think I'll have it as a date object to give the target module the option of transforming it the way they want without having to parse it.